src/containerapp/azext_containerapp/_constants.py

```diff
@@ -77,3 +77,10 @@
 HELLO_WORLD_IMAGE = "mcr.microsoft.com/k8se/quickstart:latest"
 
 LOGS_STRING = '[{"category":"ContainerAppConsoleLogs","categoryGroup":null,"enabled":true,"retentionPolicy":{"days":0,"enabled":false}},{"category":"ContainerAppSystemLogs","categoryGroup":null,"enabled":true,"retentionPolicy":{"days":0,"enabled":false}}]'  # pylint: disable=line-too-long
+
+DAPR_REDIS_PREFIX = "dapr-redis"
+DAPR_REDIS_SECRET_NAME = "redis-config"
+DAPR_REDIS_CONFIG_PASSWORD_KEY = "requirepass"
+DAPR_REDIS_PORT = 6379
+DAPR_COMPONENT_REDIS_STATESTORE_NAME = "redis-statestore"
+DAPR_COMPONENT_REDIS_PUBSUB_NAME = "redis-pubsub"
```

Comment:
We should make these match `dapr init`, which I think creates them as `statestore` and `pubsub`. That way you can go from local to ACA without code change.